personal_assistant/classes/note_taker_classes.py

```diff
@@ -1,10 +1,66 @@
 from collections import UserDict
-from personal_assistant.classes.base_classes import Field
 
 
-class Note(Field):
-    pass
+class Note:
+    def __init__(self, name="Untitled Note", tags=[], text=None):
+        self.name = name
+        self.tags = tags
+        self.text = text
+
+    def __str__(self):
+        return f"{self.name}, {self.tags}, {self.text}"
 
 
 class NoteBook(UserDict):
-    pass
+    def __init__(self):
+        super().__init__()
+
+    def add_note(self):
+        name = input("Enter note name: ")
+        tags = input("Enter note tags (comma-separated): ").split(",")
+        text = input("Enter note text: ")
+
+        note = Note(name.strip(), [tag.strip() for tag in tags], text.strip())
+        self.data[name] = note
```

Comment:
I think this dictionary needs to be renamed - it breaks contacts lookup because it finds notes as well:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/35877454-137a-4e53-8612-b23dd9444840)